config/install/core.entity_form_display.node.stanford_news.default.yml

```diff
@@ -5,13 +5,16 @@ dependencies:
     - field.field.node.stanford_news.body
     - field.field.node.stanford_news.field_s_news_category
     - field.field.node.stanford_news.field_s_news_date
-    - field.field.node.stanford_news.field_s_news_hero_banner
+    - field.field.node.stanford_news.field_s_news_feat_image
```

Comment:
@pookmish, This field name change will need some of your fancy upgrade path goodness.